src/test/java/io/reactivex/internal/operators/completable/CompletableFromRunnableTest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivex.internal.operators.completable;
+
+import io.reactivex.Completable;
+import org.junit.Test;
+
+public class CompletableFromRunnableTest {
+    @Test(expected = NullPointerException.class)
+    public void fromRunnableNull() {
+        Completable.fromRunnable(null);
+    }
+
+    @Test
+    public void fromRunnable() {
+        Completable.fromRunnable(new Runnable() {
+            @Override
+            public void run() {
+
```

Comment:
Can also test calling twice increments twice. And that it calls lazily (not on creation).